mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -502,12 +502,12 @@ func resourceMongoDBAtlasClusterCreate(d *schema.ResourceData, meta interface{})
 	}
 
 	var containerId string
-	if providerName == "AZURE"{
+	if providerName == "AZURE" {
 		if v, ok := d.GetOk("container_id"); ok {
 			containerId = v.(string)
```

Comment:
Why do you have this validation? if the `containerId` attribute is computed (you can't get the value), it never will set by the user, also it's unnecessary to create a variable `containerId` (because it will always be empty)

Reply:
I misunderstood the behavior, I'll change it, thank you paco